### PR TITLE
rna prep to arnold

### PR DESCRIPTION
**Pull request by mayabrandi**
A new RNA workflow has been created in lims and the relevant data needs to be stored in arnold for future trending.

### Added
- RNA prep step modles
- added rna to load arnold cli

### Review:
- [ ] Code approved by
- [x] Tests executed on lims-stage by @mayabrandi 25/2
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg_lims/models/arnold/prep/rna/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+from genologics.lims import Lims
+
+from .a_tailing_and_adapter_ligation import get_a_tailning_and_adapter_ligation
+from .aliquot_samples_for_fragmentation import get_aliquot_samples_for_enzymatic_fragmentation
+from .reception_control import get_sample_artifact_fields
+from .normailzation_of_samples_for_sequencing import get_normalization_of_samples
```
Comment (annaengstrom) [inline cg_lims/models/arnold/prep/rna/__init__.py:8]:
```suggestion
from .normalization_of_samples_for_sequencing import get_normalization_of_samples
```

Comment (annaengstrom) [reply cg_lims/models/arnold/prep/rna/__init__.py:8]: Just a thought if this is a mispelling?

File: cg_lims/models/arnold/prep/rna/normailzation_of_samples_for_sequencing.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import logging
```
Comment (annaengstrom) [inline cg_lims/models/arnold/prep/rna/normailzation_of_samples_for_sequencing.py:1]: Just a thought, is this script mispelled "**normailzation**" `cg_lims/models/arnold/prep/rna/normailzation_of_samples_for_sequencing.py`

File: cg_lims/models/arnold/prep/rna/__init__.py (hunk 1 above)
Comment (mayabrandi) [reply cg_lims/models/arnold/prep/rna/__init__.py:8]: 👍

File: cg_lims/models/arnold/prep/rna/normailzation_of_samples_for_sequencing.py (hunk 2 above)
Comment (mayabrandi) [reply cg_lims/models/arnold/prep/rna/normailzation_of_samples_for_sequencing.py:1]: 👍

**Pull request merged**